pkg/console/operator/sync_v400.go

```diff
@@ -82,6 +82,9 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	// The sync loop may not settle, we are unable to honor it in current state.
 	status.HandleProgressingOrDegraded(updatedOperatorConfig, "CustomLogoSync", customLogoErrReason, customLogoError)
 
+	defaultIngressCertConfigMap, defaultIngressCertErrReason, defaultIngressCertError := co.ValidateDefaultIngressCertConfigMap()
+	status.HandleProgressingOrDegraded(updatedOperatorConfig, "DefaultIngressCertValidation", defaultIngressCertErrReason, defaultIngressCertError)
```

Comment:
One thing about bringing this back,  we need to make sure we don't panic again, per this bug:

https://bugzilla.redhat.com/show_bug.cgi?id=1778930

Reply:
true, we are missing error check.. need to add:

```go
if defaultIngressCertError != nil {
    return defaultIngressCertError
}
```